manifests/init.pp

```diff
@@ -132,6 +137,9 @@
   validate_re($contextpath, ['^$', '^/.*'])
   validate_hash($resources)
   validate_hash($ajp)
+  validate_bool($tomcatNativeSsl)
+  validate_absolute_path($tomcatKeystoreFile)
+  validate_re($tomcatKeystoreType, '^(JKS|JCEKS|PKCS12)$')
```

Comment:
Not that i know of, seems something like below might work for between 1024 to 65535 and will not accept any 0000.
```
validate_re($port, '^(102[4-9]|10[3-9]\d|1[1-9]\d{2}|[2-9]\d{3}|[1-5]\d{4}|6[0-4]\d{3}|65[0-4]\d{2}|655[0-2]\d|6553[0-5])$')
```
From here: http://stackoverflow.com/questions/25450318/how-to-limit-regex-validation-for-port-number-validation-1024-to-65535